src/eos-metrics-instrumentation.c

```diff
@@ -948,7 +953,7 @@ record_network_change (GDBusProxy *dbus_proxy,
       /* schedule recording the network ID provided we have a default route */
       if (new_network_state >= NM_STATE_CONNECTED_SITE)
         {
-          g_idle_add ((GSourceFunc) record_network_id, GINT_TO_POINTER (FALSE));
+          g_idle_add ((GSourceFunc) record_network_id, NULL);
```

Comment:
Seems wrong – this will pass `NULL` as `image_version` so will never do anything, even on solutions images.

Reply:
ah, this was a bit careless. Fixed.